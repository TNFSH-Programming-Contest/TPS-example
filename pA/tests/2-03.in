4984 * 4404
