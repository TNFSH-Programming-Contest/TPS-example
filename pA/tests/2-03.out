4984 * 4404 = 21949536
